src/consensus/blockstat.go

```diff
@@ -290,11 +289,6 @@ func (self *BlockStat) try_add_hash_and_sig(
 	self.debug_count += 1
 	self.debug_usage += 1
 
-	if !(action_update || action_skip || action_insert) {
```

Comment:
Comment out this code instead (normally you should delete it, but this code in the `consensus/` folder is reference code and I do not want to remove what was there)